api/index.js

```diff
@@ -2,7 +2,7 @@ import config from './src/constants.js';
 import Map from './src/Map.js';
 
 config.themesUrl = 'https://geomapfish-demo-2-4.camptocamp.com/themes?' +
-  'version=2&background=background&interface=api';
+  'version=2&background=background&interface=desktop';
```

Comment:
This should not be committed...

Reply:
Commited by mistake, but note that interface API does not exists on demo 2.4.

Reply:
Just created :-)